app/views/posts/_full_post.html.slim

```diff
@@ -0,0 +1,67 @@
+div.container
+	div.row
+		div.col-md-3
+			.profile_container
+					img class='profilepic_small' src="#{post.user.image}?sz=200"
+					.ticket_name #{post.user}
+		div.col-md-6
+			div.short #{post.short}
+			div.tags 
+				= post.tag_list.map{ |tag| "<span class='tag'>#{ tag }</span>" }.join(", ").html_safe
+			div.timestamp 
+				- difference = time_diff post.updated_at
+				| updated #{difference}
```

Comment:
change your helper method so you can just give it a reference to the `post` and it'll return a string